### DISCUSSION TITLE
Gaussian wake model (growth rate parameter)

### DISCUSSION THREAD

**Discussion by seltzera** [NatLabRockies/floris#879] [Q&A] [answered]
Is it correctly understood that when using the "gauss" wake model in an input file, the wake growth rate is given by the equation "k*=ka*I+kb" from [1], where I is the local turbulence intensity at the turbine, and ka & kb are the variables that can be set in an input file but the default values are from [1]? 

[1]: Amin Niayifar and Fernando Porté-Agel. Analytical modeling of wind farms: a new approach for power prediction. Energies, 9(9):741, 2016.

**Comment by misi9170** [comment, marked as answer]
@seltzera , that's correct, also reported in a slightly [earlier paper from Niayifar and Porté-Agel](https://iopscience.iop.org/article/10.1088/1742-6596/625/1/012039). In the FLORIS implementation, the values are rounded to `ka = 0.38` and `kb = 0.004` (whereas in Niayifar and Porté-Agel (2015), equation (15) is the higher-precision $k^* = 0.3837I + 0.003678$, i.e., $k_a = 0.3837$ and $k_b = 0.003678$).
